redbadger/website-honestly
# Privacy policy

### Changes

#### Trello Ticket / Github Issue Link

https://trello.com/c/Q4hzp63M/183-gdpr-privacy-policy

#### Description

Add the new privacy policy page. This reuses some of the styles from the cookie policy, but does not share them (yet). As we are going to have a third page (terms & conditions) I wanted to wait and see what kind styles/components will be good as more general abstractions.

#### Screenshots

![screen shot 2018-05-24 at 17 07 31](https://user-images.githubusercontent.com/1148118/40497793-135a9f2a-5f75-11e8-8c45-57a8c44c9388.jpg)

### Test

#### Test plan

- check cross-browser styling
- check links work
- check design is correct
